install.bash

```diff
@@ -24,6 +24,7 @@ if [ ! -d "$install_dir/.sufler" ]; then
     printf "\n\e[1;31mSufler installation unsuccessful due to failed unzip\e[0m\n"
     exit
 fi
-pip install -e "$install_dir/.sufler/"
+sudo pip install -e "$install_dir/.sufler/sufler-master/"
```

Comment:
what if user doesn't want to use sudo?

Reply:
I don't know. I use sudo because that's only way to make pip install sufler package. Maybe you have any suggestions how to work around it?

Reply:
maybe you could go for adding sudo in outer scope so that it's manageable:
```
sudo bash -c "$(curl -fsSL https://raw.githubusercontent.com/limebrains/sufler/master/install.bash)"
```

Reply:
ok, i changed it